src/DashboardUI/package.json

```diff
@@ -60,13 +60,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
```

Comment:
Why don't we want this for react-app and react-app/jest?

Reply:
Config is just moved to a different file, out of package.json